docs/architecture/data-plane-osd-cluster-dynamic-scaling.md

```diff
@@ -10,138 +10,6 @@ Data Plane OSD Cluster Dynamic Scaling functionality currently deals with:
-scaled is a Multi-AZ cluster and `.status.resizeInfo.nodeDelta` is not a multiple
-of the Multi-AZ required number of nodes (3) then the value to increase will be
-round-up to that nearest multiple.
-
-(At least) for the initial version, when performing the scale-up increase, if
-that increment does not reach the ScaleUpThreshold we will still increase only
-that value. Then, on subsequent calls to the agent status report endpoint it
-will continue scaling step by step until the threshold is reached. In this way
-it will scale in a more controlled way.
-
-## Compute nodes Scale-Down criteria
-
-KAS Fleet Manager will scale down compute nodes of a data plane cluster if all
-the following conditions are true:
-
-* **All** (notice the difference with scale-up criteria) of the reported Kafka
-  attribute values have crossed their corresponding Scale-Down threshold
-* The current number of nodes is strictly higher than
-  the `.status.nodeInfo.currentWorkLoadMinimum` value
-* The number of nodes that would remain after performing the Scale-Down value
-  calculation (see subsection below) is higher
-  than `.status.nodeInfo.currentWorkLoadMinimum` , higher than restricted floor,
-  higher than multi-az multiple requirement (3) and higher
-  than 0.
-* There's no scaling action already in progress
-
-Restricted Floor: The provided floor in the
-status (`status.nodeInfo.floor`) but rounded up to multi-az node
-requirements (3) ONLY in case the cluster is multi-az. In case
-cluster isn't multi-az then restricted floor is the same as reported floor
-
-Scale-Down thresholds:
-  * Kafka Connections: The connections specified in `resizeInfo.delta.connections`.
-    Due to it is currently being assumed `resizeInfo.nodeDelta` will always be
-    3 this means that it is a value equivalent to 3 full OSD Compute nodes worth
-    of connections
-  * Kafka Partitions: The partitions specified in `resizseInfo.delta.partitions`.
-    Due to it is currently being assumed `resizeInfo.nodeDelta` will always be 3
-    this means that it is a value equivalent to 3 full OSD Compute nodes worth
-    of partitions
-
-### Compute nodes Scale-Down value calculation
-
-Once KAS Fleet Manager has decided to perform scale down of a data plane's compute
-nodes, it will try to scale down by the value specified
-in `.status.resizeInfo.nodeDelta`. If the cluster
-being scaled is a Multi-AZ cluster and `.status.resizeInfo.nodeDelta` is not a
-multiple of the Multi-AZ required number of nodes (3) then the value to
-decrease will be round-up to that nearest multiple.
-
-(At least) for the initial version, when performing the nodeDelta decrease, if
-that decrement does not reach the ScaleDownThreshold we will still decrease
-only the specified nodeDelta  value anyway. Then, on subsequent calls to the
-agent status report endpoint it will continue scaling down until the threshold
-or currentWorkLoadMin is reached. In this way it will scale in a more
-controlled way
-
```

Comment:
We should delete all content or replace it with a TODO. The sections below or above here don't apply anymore either

Reply:
Either way it is fine with me.
My intention of leaving them is that they'll have to be updated when the cluster creation / deletion logic is revised.

Reply:
:+1: